source-map.bs

```diff
@@ -97,19 +102,22 @@ urlPrefix:https://webassembly.github.io/spec/core/; type:dfn; spec:wasm
 }
 </pre>
 
-## License
+<h2 class="no-num no-toc">About this Specification</h2>
 
-This work is licensed under a [Creative Commons Attribution-ShareAlike 3.0 Unported License](http://creativecommons.org/licenses/by-sa/3.0/).
+The document at [https://tc39.es/source-map/](https://tc39.es/source-map/) is the most accurate and
+up-to-date Source Map specification. It contains the content of the most recently published snapshot
+plus any modifications that will be included in the next snapshot.
+
+If you want to get involved you will find more information under the following [Specification
+Repository](https://github.com/tc39/source-map).
 
-## Introduction
+<h2 class="no-num">Introduction</h2>
 
-This document is a draft version of a hardened version of the Source Map v3
-specification.  In its current form, it's not a defined standard and is subject to
-modifications.  If you want to get involved you will find more information under
-the following [Specification Repository](https://github.com/tc39/source-map).
+This Ecma standard defines the Source Map format, used for mapping transpiled source code back to the original sources.
 
-Background {#background}
-========================
+The Source Maps format has the following goals:
+* Support source-level debugging allowing bidirectional mapping
+* Support server-side stack trace deobfuscation
```

Comment:
Should we drop the `server-side`? It's a very loaded term and we don't provide a clear definition what we mean.

Reply:
Here I just copy-pasted this section from further below in the document.

Reply:
Ah, I missed that this was a copy-paste. Removing in a separate PR sgtm